Sources/App/Features/Activities/Controllers/ActivityAPIController.swift

```diff
@@ -0,0 +1,102 @@
+import Vapor
+import Fluent
+import S3
+
+struct ActivityAPIController: RouteCollection {
+    private struct FormInput: Content {
+        let eventID: String
+        let title: String
+        let subtitle: String?
+        let description: String?
+        let metadataURL: String?
+        var activityImage: File
+    }
+
+    func boot(routes: RoutesBuilder) throws {
+        routes.post("", use: onPost)
```

Comment:
What does this route look like?

Reply:
Will be the similar path structure as the routes on presentations and slots. Empty `""` is the path defined at the higher level up.

```
+------+------------------------------+
| POST | /api/v1/presentations/       |
+------+------------------------------+
| POST | /api/v1/presentations/:id    |
+------+------------------------------+
| POST | /api/v1/slots/               |
+------+------------------------------+
| POST | /api/v1/slots/:id            |
+------+------------------------------+
| POST | /api/v1/activities/          |
+------+------------------------------+
| POST | /api/v1/activities/:id       |
+------+------------------------------+
```